src/components/Button/__tests__/index-test.js

```diff
@@ -1,5 +1,52 @@
 /* eslint-env jasmine, jest */
 
+import React from 'react';
+import Button from '..';
+import { mount, shallow } from 'enzyme';
+
+const findNativeButton = wrapper => wrapper.find('[role="button"]');
+
 describe('components/Button', () => {
-  test.skip('NO TEST COVERAGE', () => {});
+  describe('rendered element', () => {
+    test('is a "div"', () => {
+      const onPress = () => {}
+      const button = findNativeButton(mount(<Button onPress={onPress} title="" />));
+      expect(button.type()).toBe('div');
+    });
+
+    test('can display text', () => {
+      const onPress = () => {}
+      const text = 'Click me';
+      const component = mount(<Button onPress={onPress} title={text} />);
+      expect(component.text()).toEqual(text);
+    });
+  });
+
+  test('prop "color"', () => {
+    const onPress = () => {}
+    const color = 'blue';
+    const button = findNativeButton(mount(<Button color={color} onPress={onPress} title="" />));
+    expect(button.prop('style')).toEqual({backgroundColor: color});
+  });
+
+  test('prop "disabled"', () => {
```

Comment:
Haha nope, not really. I could update it to test that the `disabled` prop prevents the button from being pressed:

```javascript
  test('prop "disabled" should prevent "onPress" from firing', () => {
    const onPress = jest.fn();
    const component = shallow(<Button disabled={true} onPress={onPress} title="" />);
    component.simulate('press');
    expect(onPress).not.toHaveBeenCalled();
  });
```

however I'm getting this failure when I do that: 

```
expect(jest.fn()).not.toHaveBeenCalled()

    Expected mock function not to be called but it was called with:
      Array []
```

Could this be an issue with simulating press events in enzyme?